### ISSUE TITLE
Missing configure after git clone https://github.com/BitchX/BitchX1.3

### ISSUE THREAD

**Issue by antdude** [BitchX/BitchX1.3#7]
Was it removed?

**Comment by antdude** [comment]
Nope. That's new to me. I never knew autogen made configure. Compiling with gmake right now. I hope this works!

**Comment by antdude** [comment]
Thanks. That works. :) Now, waiting for OpenSSL issue to be fixed. ;)

**Comment by JonnyWhatshisface** [comment]
Cool. Please close the issue out so it doesn't sit around :) 

OpenSSL Automake patch was merged yesterday, so upstream should now compile for you.

**Comment by antdude** [comment]
Yes, it compiled and worked. Thanks. ;)
